### DISCUSSION TITLE
Today I lost 2 PUCS again.

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#13942] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

I think if I save the game, then my items should be preserved.
Today I lost 2 PUCS again.
Because the bots abandoned them at the station, and I saved and exit instead of dock off.
What is the point of saving then? If the items left at the station disappear?

### Reproduction steps

Drop PUCS on station.
Then save and exit.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
[You have already reported this](https://github.com/FakeFishGames/Barotrauma/discussions/13652), closing as a duplicate.

**Comment by itchyOwl** [comment]
I'm sorry for your loss, but it's not intentional and should have been fixed. If it still happens, we should reopen the original ticket or create a new. Reopening this now, because the original was closed as solved.

**Comment by itchyOwl** [comment]
As a clarification: if you drop the suits, they are not teleported back to the sub. It only happens if the bots do that.

**Comment by Regalis11** [comment]
@itchyOwl  [The fix](https://github.com/FakeFishGames/Barotrauma-development/commit/b1d511ac1c50fd60eb23620ee74bb9c18cac72d7) is not yet included in the public version of the game, so I believe this is a duplicate that should be closed.
